index.js

```diff
@@ -4,7 +4,7 @@ import chalk from "chalk";
 
 export async function main(argv = {}) {
 	try {
-		console.log(chalk.blue("\n🚀 Starting AI Commit Generator..."));
```

Comment:
Two messages were logged on initiating `commitg`. I have replaced this message with the one that came after.